examples/vanilla-ts-esm/public/mux-uploader-simple.html

```diff
@@ -53,23 +53,30 @@ <h1><a href="/">Elements</a></h1>
 
     <h2>Enter your upload GCS url:</h2>
     <div>
-      <input type="text" class="direct-upload-url" placeholder="https://storage.googleapis.com/..." />
+      <input type="text" class="direct-upload-url" placeholder="https://storage.googleapis.com/..."/>
     </div>
 
     <div style="margin: 40px 0;">
-        <mux-uploader></mux-uploader>
+        <!-- You can remove `pausable` attribute if you don't want to expose that UI.  -->
+        <mux-uploader pausable></mux-uploader>
     </div>
 
     <a href="../">Browse Elements</a>
 
     <script>
       const input = document.querySelector('input[type="text"]');
       const muxUploader = document.querySelector('mux-uploader');
-      input.addEventListener('change', () => {
+      const updateEndpoint =  () => {
```

Comment:
NOTE: changed this to make it easier to dump a `value` in the input while testing locally.